aten/src/ATen/native/quantized/TensorAdvancedIndexing.cpp

```diff
@@ -182,9 +182,9 @@ Tensor& _index_put_impl_quantized_cuda_(Tensor & self, const torch::List<std::op
 
   at::assert_no_overlap(self, value);
   // NOLINTNEXTLINE(performance-implicit-conversion-in-loop)
-  for (const std::optional<Tensor>& index: indices) {
-    if (index.has_value()) {
-      at::assert_no_overlap(self, *index);
+  for (const auto index: indices) {
+    if (index.get().isTensor()) {
```

Comment:
It's late, and I might not know how to read C++, but `std::optional<>` does not have a method `.get()` does it?